spec/models/rules/basic_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe Rules::Basic, type: :model do
```

Comment:
we shouldn't have to specify type bc we have `config.infer_spec_type_from_file_location!` in `rails_helper`